src/MessagePassing/loopcorrection.jl

```diff
@@ -63,34 +64,49 @@ function sim_edgeinduced_subgraph(bpc::BeliefPropagationCache, eg)
 end
 
 #Get the all edges incident to the region specified by the vector of edges passed
+# Optionally pass in a vector of vertices in the region, to handle the corner case where the region is just one vertex. 
```

Comment:
If you region is vertex based instead, this can be handled by `boundary_edges(bpc::BeliefPropagationCache, verts::Vector)` which should be an existing function (as there is a `NamedGraphs` version of it)